compose/ui/ui/src/jsWasmMain/kotlin/androidx/compose/ui/window/ComposeWindow.js.kt

```diff
@@ -159,6 +180,7 @@ private class ComposeWindow(
             event.preventDefault()
         })
 
+        // TODO Shouldn't we use e.preventDefault() to replace browser`s hotkeys(Tab, Ctrl+S, etc)?
```

Comment:
Please, avoid adding comments to PR just to discuss any - however valid - topics, it's just the wrong tool for the purpose